pkg/tpm/tpm1.go

```diff
@@ -157,10 +180,11 @@ func (t *TPM1) Close() {
 // Summary returns a string with formatted TPM information
 func (t TPM1) Summary() string {
```

Comment:
as discussed in an unrelated diff (in linuxboot/fiano), `Summary` is suboptimal and we'd better use JSON and (if necessary) introspection. I agree with that, and we can implement it in the next diff. Then we can remove Summary entirely